### PR TITLE
Update CI

### PR DESCRIPTION
**Pull request by goosemania**
[noissue]

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -61,8 +61,6 @@ jobs:
       matrix:
         env:
           - TEST: pulp
-          - TEST: docs
-          - TEST: s3
```
Comment (goosemania) [inline .github/workflows/ci.yml:65]: Why are those removed?

Comment (daviddavis) [reply .github/workflows/ci.yml:65]: Because `test_s3` and `docs_test` are `false` in your template_config.yml. There was a bug that got fixed where these tests weren't being properly disabled.

**Review by ipanova — APPROVED** [review]
